added docs restructuring (#81)
dappforce/subsocial-docs: docs/develop/SDKCheatsheet.md

  }
});
```

> Read more about signing & sending transaction [here](/docs/develop/sdk/transactions)

Reviewer: ```suggestion
Read more about signing & sending transactions [here](/docs/develop/sdk/transactions).
```

  </TabItem>
</Tabs>

Check full docs [here](/docs/develop/sdk/profiles/fetch-profiles).

## Writing Data

### Note

To store data on IPFS, it is necessary to setup CRUST IPFS account and pushing the data from your account. 

You set authHeader and setup with Subsocial SDK like this:
> This is only required for Testnet. On Mainnet, this will not work.

```js

> import { generateCrustAuthToken } from "@subsocial/api/utils/ipfs";

Reviewer 2: Remove the unused import

> const authHeader = 'c3ViLTVGQTluUURWZzI2N0RFZDhtMVp5cFhMQm52TjdTRnhZd1Y3bmRxU1lHaU45VFRwdToweDEwMmQ3ZmJhYWQwZGUwNzFjNDFmM2NjYzQzYmQ0NzIxNzFkZGFiYWM0MzEzZTc5YTY3ZWExOWM0OWFlNjgyZjY0YWUxMmRlY2YyNzhjNTEwZGY4YzZjZTZhYzdlZTEwNzY2N2YzYTBjZjM5OGUxN2VhMzAyMmRkNmEyYjc1OTBi';

Reviewer 2: Let's use it in all places instead of generateCrustAuthToken.

  </TabItem>
</Tabs>

Check full docs [here](/docs/develop/sdk/profiles/fetch-profiles).

## Writing Data

### Note

> To store data on IPFS, it is necessary to setup CRUST IPFS account and pushing the data from your account. 

Reviewer: ```suggestion
To store data on IPFS, it is necessary to set up a CRUST IPFS account and push the data from your account. 
```

You set authHeader and setup with Subsocial SDK like this:

> > This is only required for Testnet. On Mainnet, this will not work.

Reviewer: ```suggestion
> This is only required for the Testnet. On the Mainnet, this will not work.
```

  </TabItem>
</Tabs>

Sign and send the transaction, [Check Here](/docs/develop/sdk/transactions)

> Check full docs [here](/docs/develop/sdk/spaces/create-spaces).

Reviewer: ```suggestion
Check the full docs [here](/docs/develop/sdk/spaces/create-spaces).
```

  </TabItem>
</Tabs>

Sign and send the transaction, [Check Here](/docs/develop/sdk/transactions)

> Check full docs [here](/docs/develop/sdk/posts/create-posts).

Reviewer: ```suggestion
Check the full docs [here](/docs/develop/sdk/posts/create-posts).
```

:::info

> Now, create a new space as mentioned [here](http://docs.subsocial.network/docs/develop/sdk-cheatsheet#space-1). So we can mark it as profile.

Reviewer: ```suggestion
Now, create a new space as mentioned [here](http://docs.subsocial.network/docs/develop/sdk-cheatsheet#space-1), so we can mark it as a profile.
```

> To change profile data, update the profile space from it's id.

Sign and send the transaction, [Check Here](/docs/develop/sdk/transactions)

> Check full docs [here](/docs/develop/sdk/profiles/create-profiles).

Reviewer: ```suggestion
Check the full docs [here](/docs/develop/sdk/profiles/create-profiles).
```

  </TabItem>
</Tabs>

> Check full docs [here](/docs/develop/sdk/comments/fetch-comments).

Reviewer: ```suggestion
Check the full docs [here](/docs/develop/sdk/comments/fetch-comments).
```

  </TabItem>
</Tabs>

> Check full docs [here](/docs/develop/sdk/follow/fetch-follow).

Reviewer: ```suggestion
Check the full docs [here](/docs/develop/sdk/follow/fetch-follow).
```

  </TabItem>
</Tabs>

Sign and send the transaction, [Check Here](/docs/develop/sdk/transactions)

> Check full docs [here](/docs/develop/sdk/follow/create-follow).

Reviewer: ```suggestion
Check the full docs [here](/docs/develop/sdk/follow/create-follow).
```

  </TabItem>
</Tabs>

Sign and send the transaction, [Check Here](/docs/develop/sdk/transactions)

> Check full docs [here](/docs/develop/sdk/reactions/create-reactions).

Reviewer: ```suggestion
Check the full docs [here](/docs/develop/sdk/reactions/create-reactions).
```
